gate-web/src/main/groovy/com/netflix/spinnaker/gate/controllers/ManagedController.java

```diff
@@ -135,17 +136,16 @@ Map getApplicationDetails(
     return keelService.getApplicationDetails(application, includeDetails);
   }
 
-  @ApiOperation(value = "Pass a message to a veto plugin", response = Map.class)
-  @PostMapping(path = "/vetos/{name}")
-  void passVetoMessage(
-      @PathVariable("name") String name, @RequestBody Map<String, Object> message) {
-    keelService.passVetoMessage(name, message);
+  @ApiOperation(value = "Pause management of an entire application")
+  @PostMapping(path = "/application/{application}/pause")
+  void pauseApplication(@PathVariable("application") String application) {
+    keelService.pauseApplication(application, Collections.emptyMap());
```

Comment:
this is silly and i don't like it, but keel is asking for a `POST` and will complain/fail if we try to make a request without anything in it. At least this prevents us from needing to expose that quirkiness to gate consumers.